lib/organize.js

```diff
@@ -24,7 +31,13 @@ module.exports.organize = (
 	const languageService = getLanguageService(parser, filepath, code);
 
 	const fileChanges = languageService.organizeImports(
-		{ type: 'file', fileName: filepath, skipDestructiveCodeActions: organizeImportsSkipDestructiveCodeActions },
+		{
+			type: 'file',
+			fileName: filepath,
+			skipDestructiveCodeActions: organizeImportsSkipDestructiveCodeActions,
+			// @ts-ignore
```

Comment:
Same here, why a ts-ignore?

Reply:
It's the problem of use enum type in javascript code, since the `languageService.organizeImports` only accept `mode` type as [`OrganizeImportsMode`](https://github.com/microsoft/TypeScript/blob/main/src/services/types.ts#L725-L729), we must use something like `OrganizeImportsMode.All` to configure it, and it was imported from the typescript code, that's not a good idea. I couldn't find any other way to solve this problem, the simple way is just ignore it.

Reply:
ok we can just type case it instead. i'll try to finish this PR soon, sorry somehow was too busy lately 🫣